core/engine/src/context/intrinsics.rs

```diff
@@ -29,15 +32,16 @@ pub struct Intrinsics {
 }
 
 impl Intrinsics {
-    pub(crate) fn new(root_shape: &RootShape) -> Self {
+    /// Creates a new set of uninitialized intrinsics.
+    pub(crate) fn uninit(root_shape: &RootShape) -> Option<Self> {
```

Comment:
thought: should a note be added to redirect anyone navigating the repo.

The API change makes sense to me, but if someone is navigating the repository for the first time. It could be useful to have a note to redirect where to look.

Reply:
Not sure what this means. Can you explain  a bit what exactly I should link?

Reply:
I meant it more of like an internal note for anyone trying familiarize themselves with the codebase. Something like:

```rust
// NOTE: Intrinsic objects are initialized by `Realm::Initialize`.  Individual initialization logic for each built-ins
// is located in their respective file `src/builtins/{object}/`.
```

Granted this could also always just be worked into the new site's contributor docs.